.github/workflows/pull-request.yaml

```diff
@@ -10,7 +10,7 @@ on:
 jobs:
     main:
         name: Validate PR title
-        runs-on: ubuntu-latest
+        runs-on: oss-4-core-runner
```

Comment:
maybe we should leave these to be on ubuntu-latest? We don't have to pay for these minutes

Reply:
reverted